File: 1_iniciante_beginner/c++/1002.cpp
/**
 * @source Beecrowd
 * @problem 1002 - Area of a Circle
 * @subject Sequential
 *
 * @author Letícia Cruz
 * @date 2025-08-25
 */

#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;

int main()
{

    const double PI = 3.14159;
    double raio, area;

    cin >> raio;

    area = PI * pow(raio, 2.0);
    cout << "A=" << setprecision(4) << fixed << area << endl;

    return 0;
}

File: 1_iniciante/R/1002.r
#' @source   Beecrowd
#' @problem  1002 - Area of a Circle
#' @subject  Sequential
#'
#' @author   Letícia Cruz
#' @date     2025-08-15

input <- file('stdin', 'r')
raio <- as.double(readLines(input, n=1)) # tranforma entrada em double

π = 3.14159

area <- π * (raio ^ 2) # usa ^ para potência

# cat não coloca espaço entre os itens: A=valor 
cat(sprintf("A=%.4f\n", area)) # sprintf para formatação das casas decimais

# cat mais usado para texto em console e write para escrever em arquivos (geralmente dados estruturados)